cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -406,6 +407,8 @@ public synchronized void updateProperties(Id.Namespace namespaceId, NamespaceMet
     }
 
     nsStore.update(builder.build());
+    // refresh the cache with new meta
+    namespaceMetaCache.refresh(namespaceId.toEntityId());
```

Comment:
Will `DefaultNamespaceAdmin` and the `DefaultNamepsaceQueryAdmin` binding creating different instances?